### PR TITLE
Add authority attestation functionality (NIP-68)

### PR DESCRIPTION
**Pull request by jklein24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +70,34 @@ $ uma-auth-cli lookup \
...
+
+#### Attesting to an app's identity
+
+First, find the app 13195 event ID by using the lookup command. Then, you can attest to the app's identity:
+
+```bash
+$ uma-auth-cli attest \
+--nsec nsec1mqxnulkqkcv0gc0dfrxz5kz7d3h665ve2dhjkrj8jmmxwm4st2zsjv2n5l \
+--relay wss://nos.lol \
+--app-pubkey npub13msd7fakpaqerq036kk0c6pf9effz5nn5yk6nqj4gtwtzr5l6fxq64z8x5 \
```
Comment (shreyav) [inline README.md:86]: nit `--app-npub`

File: README.md [hunk 2]
```diff
@@ -70,3 +70,34 @@ $ uma-auth-cli lookup \
...
+- `--relay` is the relay to use to publish the attestation.
+- `--app-npub` is the clients app's public key.
+- `--event-id` is the event ID of the app's 13195 event.
+
+#### Looking up attestations
+
+You can look up attestations using the CLI:
+
+```bash
+$ uma-auth-cli attestations \
```
Comment (shreyav) [inline README.md:100]: hmm should it be `lookup`?

**Review by shreyav — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jklein24) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (jklein24) [reply README.md]: Done

**Pull request merged**